nodogsplash2/files/nodogsplash.config

```diff
@@ -54,3 +60,8 @@ config nodogsplash
 
   #MAC addresses that do not need to authenticate
   #list trustedmac '00:00:C0:01:D0:1D'
+
+  # Set FW_MARK for compatibilty with other OpenWrt Packages eg mwan3, sqm etc.
+  list fw_mark_authenticated '520'
+  list fw_mark_trusted '510'
+  list fw_mark_blocked '500'
```

Comment:
Did this change help and work?

Reply:
In Chaos Calmer it was required for both MWAN3 and SQM-scripts.
In LEDE 17.01.04 it should allow MWAN3 and NDS to work together but there must be something else happening as mentioned elsewhere. We will still need it when the other problems are found.
However:
It is definitely required for SQM to work with NDS on 170104 when bound to the same interface as NDS (This gives client by client traffic control as well as speeding up wifi).